feat: improve theming guide (#29)
onecx/docs-guides-ui: docs/modules/angular/pages/cookbook/primeng/theming.adoc

= OneCX PrimeNG Theming

:idprefix:
:idseparator: -

[#overview]
== Overview

> In OneCX PrimeNG theming is handled by the OneCX shell in collaboration with OneCX Theme Management application.

Reviewer: ```suggestion
In OneCX PrimeNG theming is handled by the OneCX shell in collaboration with the OneCX Theme Management application.
```

OneCX compatible application:

* can influence PrimeNG design token values for itself.

[#theme-overrides]
== Theme Overrides
In OneCX it is possible to style the PrimeNG content differently for each Application. The mechanism that allows that is called theme overrides. Each application can specify what overrides should be applied to style PrimeNG content differently than in other applications on the page.

[#appropriate-scenarios]
=== Appropriate Scenarios
This feature should be used when the application wants to control the style of:

* PrimeNG components
* OneCX components that base on PrimeNG components

[#usage-instructions]
=== Usage Instructions

> To use this feature, it's required to add the options object as an argument to the `provideThemeConfig` call. This argument should have a property called `overrides` that can be one of the following:

Reviewer: ```suggestion
To use this feature, it's required to add the options object as an argument to the `provideThemeConfig` call. This argument has a property called `overrides` that can be one of the following:
```
